### PR TITLE
Fix user sync track

### PR DESCRIPTION
**Pull request by jefmoura**
## Purpose
The auth pipeline should trigger the creation of a user in TolaTrack as the registration form does.

Related ticket: #918

### CODE REVIEW

File: tola/auth_pipeline.py [hunk 1]
```diff
@@ -26,6 +28,20 @@ def user_to_tola(backend, user, response, *args, **kwargs):
         userprofile.email = response.get('emails["value"]')
         userprofile.save()
 
+        generated_pass = '%032x' % random.getrandbits(128)
```
Comment (Menda) [inline tola/auth_pipeline.py]: What is `%032x`?

File: tola/util.py [hunk 2]
```diff
@@ -116,3 +119,23 @@ def in_groups(u):
             raise PermissionDenied
         return False
     return user_passes_test(in_groups)
+
+
+def register_in_track(data, tolauser):
+        headers = {
```
Comment (Menda) [inline tola/util.py]: 8 spaces

File: tola/util.py [hunk 3]
```diff
@@ -116,3 +119,23 @@ def in_groups(u):
             raise PermissionDenied
         return False
     return user_passes_test(in_groups)
+
+
+def register_in_track(data, tolauser):
```
Comment (Menda) [inline tola/util.py]: I would avoid to create this function in `util.py`, and would create a new file for this - e.g.: `registration.py`, `registration_bl.py` (bl = Business Logic), `services.py`, or any other better name that you come up with. The reason is that we are going to move to a decoupled package from the CE, and we don't want to use any of the util function. Also, if for easiness we decide to move that function, we just need to copy the file.

File: tola/auth_pipeline.py (hunk 1 above)
Comment (jefmoura) [reply tola/auth_pipeline.py]: Formatting string. Have a look at the documentation. https://docs.python.org/2/library/string.html

File: tola/util.py (hunk 2 above)
Comment (jefmoura) [reply tola/util.py]: Good eye!

File: tola/util.py (hunk 3 above)
Comment (jefmoura) [reply tola/util.py]: Moving to registration? Okay... it makes sense but we should also move the RegisterView and other functions related to the registration.

**Pull request merged**